src/components/registerForm/RegisterForm.js

```diff
@@ -0,0 +1,20 @@
+import styles from "@/styles/logInModal.module.scss";
+
+const RegisterForm = ({setToLogin,setToAnonymous}) => {
+  return (
+    <>
+      <h3 onClick={setToLogin} className={styles.logInModalLink}>Logga in</h3>
+      <h2>Registrera ny användare</h2>
+      <form>
+        <input required placeholder="Användarnamn" type="text"></input>
+        <input required placeholder="Email" type="email"></input>
```

Comment:
For accessibility reasons, all input fields should have labels. Screen readers won't read placeholders, so those users have no way of knowing what any field without labels is for. 

Some reading: https://medium.com/codex/5-ways-to-make-labels-accessible-for-input-controls-and-3-ways-not-to-55ecb2a2e80a

Reply:
👍 I will add labels for all input elements.